### ISSUE TITLE
[web] Canvas Kit issue when using dart:ui/Path.from - BindingError: Tried to invoke ctor of Path with invalid number of parameters (1) - expected (0) parameters instead!

### ISSUE THREAD

**Issue by emmett-deen** [flutter/flutter#75473]
I ran into this issue when using the fl_chart package using the newest beta branch. This appears to be a Flutter issue & not a package issue. When I was digging through the updates to the beta branch I noticed that CanvasKit was recently updated which I suspect is the cause of this issue. When running the application using the HTML web renderer the package runs without issues.

HTML rendering run command used:
`flutter run -d chrome --web-renderer html`

```
Flutter 1.26.0-17.2.pre • channel beta • https://github.com/flutter/flutter.git
Framework • revision 79b49b9e10 (27 hours ago) • 2021-02-03 15:33:39 -0800
Engine • revision 2c527d6c7e
Tools • Dart 2.12.0 (build 2.12.0-259.8.beta)
```

Note this was also tested on the most recent master branch

## Steps to Reproduce

`Path.from(Path() /*this path could be any path*/)`;

**Expected results:** A clone of the source path

**Actual results:** 
`BindingError: Tried to invoke ctor of Path with invalid number of parameters (1) - expected (0) parameters instead!
`

<details>
  <summary>Logs</summary>

```
BindingError: Tried to invoke ctor of Path with invalid number of parameters (1) - expected (0) parameters instead!
https://unpkg.com/canvaskit-wasm@0.22.0/bin/canvaskit.js 153:76        <fn>
https://unpkg.com/canvaskit-wasm@0.22.0/bin/canvaskit.js 152:335       new BindingError
https://unpkg.com/canvaskit-wasm@0.22.0/bin/canvaskit.js 206:164       <fn>
https://unpkg.com/canvaskit-wasm@0.22.0/bin/canvaskit.js 152:335       new Path
[38;5;248mlib/_engine/engine/canvaskit/path.dart 16:15                           from[39;49m
[38;5;248mlib/ui/src/ui/path.dart 18:28                                          from[39;49m
[38;5;248mpackages/fl_chart/src/chart/line_chart/line_chart_painter.dart 485:33  [_generateBelowBarPath][39;49m
[38;5;248mpackages/fl_chart/src/chart/line_chart/line_chart_painter.dart 221:11  [_drawBarLine][39;49m
[38;5;248mpackages/fl_chart/src/chart/line_chart/line_chart_painter.dart 109:7   paint[39;49m
[38;5;248mpackages/flutter/src/rendering/custom_paint.dart 564:12                [_paintWithPainter][39;49m
[38;5;248mpackages/flutter/src/rendering/custom_paint.dart 605:7                 paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/proxy_box.dart 142:14                   paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/proxy_box.dart 142:14                   paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/proxy_box.dart 142:14                   paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/shifted_box.dart 72:14                  paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/box.dart 2805:14                        defaultPaint[39;49m
[38;5;248mpackages/flutter/src/rendering/flex.dart 1079:7                        paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/box.dart 2805:14                        defaultPaint[39;49m
[38;5;248mpackages/flutter/src/rendering/stack.dart 629:5                        paintStack[39;49m
[38;5;248mpackages/flutter/src/rendering/stack.dart 639:7                        paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/proxy_box.dart 142:14                   paint[39;49m
[38;5;248mpackages/flutter/src/rendering/proxy_box.dart 2165:11                  paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/proxy_box.dart 142:14                   paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/shifted_box.dart 72:14                  paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 189:12                      paintChild[39;49m
[38;5;248mpackages/flutter/src/rendering/proxy_box.dart 142:14                   paint[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 2322:7                      [_paintWithContext][39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 140:10                      _repaintCompositedChild[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 100:5                       repaintCompositedChild[39;49m
[38;5;248mpackages/flutter/src/rendering/object.dart 978:29                      flushPaint[39;49m
[38;5;248mpackages/flutter/src/rendering/binding.dart 455:19                     drawFrame[39;49m
[38;5;248mpackages/flutter/src/widgets/binding.dart 874:13                       drawFrame[39;49m
[38;5;248mpackages/flutter/src/rendering/binding.dart 319:5                      [_handlePersistentFrameCallback][39;49m
[38;5;248mpackages/flutter/src/scheduler/binding.dart 1144:15                    [_invokeFrameCallback][39;49m
[38;5;248mpackages/flutter/src/scheduler/binding.dart 1082:9                     handleDrawFrame[39;49m
[38;5;248mpackages/flutter/src/scheduler/binding.dart 998:5                      [_handleDrawFrame][39;49m
[38;5;248mlib/_engine/engine/platform_dispatcher.dart 896:13                     invoke[39;49m
[38;5;248mlib/_engine/engine/platform_dispatcher.dart 145:5                      invokeOnDrawFrame[39;49m
[38;5;248mlib/_engine/engine.dart 259:45                                         <fn>[39;49m
```

```
```

```
```

</details>

**Comment by TahaTesser** [comment]
Hi @emmett-deen 
Can you please provide  a complete reproducible minimal code sample
Thank you

**Comment by emmett-deen** [comment]
@TahaTesser here is a small application that throws the error. Please note when running with HTML as the renderer there are no issues. The issue is only with using CanvasKit as the rendering engine.

Works - `flutter run -d chrome --web-renderer html`
Crashes - `flutter run -d chrome --web-renderer canvaskit`

<details>
<summary>
Sample Application:
</summary>

```
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Path.from bug example',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: ExamplePage(),
    );
  }
}

class ExamplePage extends StatefulWidget {
  ExamplePage({Key key}) : super(key: key);

  @override
  _ExamplePageState createState() => _ExamplePageState();
}

class _ExamplePageState extends State<ExamplePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: _buildBody(),
    );
  }

  Widget _buildBody() {
    return CustomPaint(
      size: MediaQuery.of(context).size,
      painter: ExamplePainter(),
    );
  }
}

class ExamplePainter extends CustomPainter {
  @override
  void paint(Canvas canvas, Size size) {
    Path path1 = new Path();
    path1.moveTo(0, 0);
    path1.lineTo(size.width / 2, size.height / 2);

    Path path2 = Path.from(path1);

    path2.lineTo(0, size.height);

    Paint paint = Paint()
      ..color = Colors.blue
      ..strokeWidth = 10;

    canvas.drawPath(path2, paint);
  }

  @override
  bool shouldRepaint(covariant CustomPainter oldDelegate) {
    return true;
  }
}

```
</details>

**Comment by TahaTesser** [comment]
Hi @emmett-deen 
Thanks for filing the issue, I can reproduce on the `beta`, `dev`, and `master` in canvaskit, not in HTML (not sure if it is a regression.) 

<details>
<summary>code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Path.from bug example',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: ExamplePage(),
    );
  }
}

class ExamplePage extends StatefulWidget {
  ExamplePage({Key key}) : super(key: key);

  @override
  _ExamplePageState createState() => _ExamplePageState();
}

class _ExamplePageState extends State<ExamplePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: _buildBody(),
    );
  }

  Widget _buildBody() {
    return CustomPaint(
      size: MediaQuery.of(context).size,
      painter: ExamplePainter(),
    );
  }
}

class ExamplePainter extends CustomPainter {
  @override
  void paint(Canvas canvas, Size size) {
    Path path1 = new Path();
    path1.moveTo(0, 0);
    path1.lineTo(size.width / 2, size.height / 2);

    Path path2 = Path.from(path1);

    path2.lineTo(0, size.height);

    Paint paint = Paint()
      ..color = Colors.blue
      ..strokeWidth = 10;

    canvas.drawPath(path2, paint);
  }

  @override
  bool shouldRepaint(covariant CustomPainter oldDelegate) {
    return true;
  }
}

```

</details>





<details>
<summary>logs</summary>

```bash
Launching lib/main.dart on Chrome in debug mode...
Waiting for connection from debug service on Chrome...             16.5s
Debug service listening on ws://127.0.0.1:64326/ipmf5N65Lcg=/ws

Running with unsound null safety
For more information see https://dart.dev/null-safety/unsound-null-safety

Warning: Flutter's support for web development is not stable yet and hasn't
been thoroughly tested in production environments.
For more information see https://flutter.dev/web

🔥  To hot restart changes while running, press "r" or "R".
For a more detailed help message, press "h". To quit, press "q".
══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
The following NativeJavaScriptObject object was thrown during paint():
  BindingError: Tried to invoke ctor of Path with invalid number of parameters (1) - expected (0)
parameters instead!

The relevant error-causing widget was:
  CustomPaint file:///Users/tahatesser/AndroidStudioProjects/beta_flutter/lib/main.dart:37:12

When the exception was thrown, this was the stack:
https://unpkg.com/canvaskit-wasm@0.22.0/bin/canvaskit.js 153:76          <fn>
https://unpkg.com/canvaskit-wasm@0.22.0/bin/canvaskit.js 152:335         new BindingError
https://unpkg.com/canvaskit-wasm@0.22.0/bin/canvaskit.js 206:164         <fn>
https://unpkg.com/canvaskit-wasm@0.22.0/bin/canvaskit.js 152:335         new Path
lib/_engine/engine/canvaskit/path.dart 16:15                             from
lib/ui/src/ui/path.dart 18:28                                            from
packages/triage/main.dart 51:23                                          paint
packages/flutter/src/rendering/custom_paint.dart 564:12                  [_paintWithPainter]
packages/flutter/src/rendering/custom_paint.dart 605:7                   paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/box.dart 2805:14                          defaultPaint
packages/flutter/src/rendering/custom_layout.dart 412:5                  paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/material/material.dart 551:11                       paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 396:12                        pushLayer
packages/flutter/src/rendering/proxy_box.dart 1925:14                    paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10                        _repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5                         repaintCompositedChild
packages/flutter/src/rendering/object.dart 206:7                         [_compositeChild]
packages/flutter/src/rendering/object.dart 187:7                         paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/box.dart 2805:14                          defaultPaint
packages/flutter/src/rendering/stack.dart 629:5                          paintStack
packages/flutter/src/rendering/stack.dart 639:7                          paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/proxy_box.dart 2165:11                    paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/proxy_box.dart 2715:13                    paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/proxy_box.dart 2715:13                    paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10                        _repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5                         repaintCompositedChild
packages/flutter/src/rendering/object.dart 206:7                         [_compositeChild]
packages/flutter/src/rendering/object.dart 187:7                         paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/proxy_box.dart 3375:11                    paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/widgets/overlay.dart 780:14                         paintStack
packages/flutter/src/widgets/overlay.dart 792:7                          paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/custom_paint.dart 608:11                  paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/view.dart 200:14                          paint
packages/flutter/src/rendering/object.dart 2322:7                        [_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10                        _repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5                         repaintCompositedChild
packages/flutter/src/rendering/object.dart 978:29                        flushPaint
packages/flutter/src/rendering/binding.dart 455:19                       drawFrame
packages/flutter/src/widgets/binding.dart 874:13                         drawFrame
packages/flutter/src/rendering/binding.dart 319:5                        [_handlePersistentFrameCallback]
packages/flutter/src/scheduler/binding.dart 1144:15                      [_invokeFrameCallback]
packages/flutter/src/scheduler/binding.dart 1082:9                       handleDrawFrame
packages/flutter/src/scheduler/binding.dart 865:7                        <fn>
dart-sdk/lib/_internal/js_dev_runtime/private/isolate_helper.dart 48:19  internalCallback

The following RenderObject was being processed when the exception was fired: RenderCustomPaint#d428e relayoutBoundary=up1:
  creator: CustomPaint ← _BodyBuilder ← MediaQuery ← LayoutId-[<_ScaffoldSlot.body>] ←
    CustomMultiChildLayout ← AnimatedBuilder ← DefaultTextStyle ← AnimatedDefaultTextStyle ←
    _InkFeatures-[GlobalKey#db86f ink renderer] ← NotificationListener<LayoutChangedNotification> ←
    PhysicalModel ← AnimatedPhysicalModel ← ⋯
  parentData: offset=Offset(0.0, 0.0); id=_ScaffoldSlot.body (can use size)
  constraints: BoxConstraints(0.0<=w<=1200.0, 0.0<=h<=752.0)
  size: Size(1200.0, 752.0)
This RenderObject has no descendants.
════════════════════════════════════════════════════════════════════════════════════════════════════

```

</details>

<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel beta, 1.26.0-17.3.pre, on macOS 11.2 20D64 darwin-x64, locale en-GB)
    • Flutter version 1.26.0-17.3.pre at /Users/tahatesser/Code/flutter_beta
    • Framework revision 4b50ca7f7f (3 days ago), 2021-02-04 19:44:27 -0800
    • Engine revision 2c527d6c7e
    • Dart version 2.12.0 (build 2.12.0-259.8.beta)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.
[!] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.
[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
[✓] VS Code (version 1.53.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0
[✓] Connected device (2 available)
    • Taha’s iPad (mobile) • 00008020-000255113EE8402E • ios            • iOS 14.4
    • Chrome (web)         • chrome                    • web-javascript • Google Chrome 88.0.4324.146
! Doctor found issues in 2 categories.
```
```bash
[✓] Flutter (Channel dev, 1.26.0-17.2.pre, on macOS 11.2 20D64 darwin-x64, locale en-GB)
    • Flutter version 1.26.0-17.2.pre at /Users/tahatesser/Code/flutter_dev
    • Framework revision 79b49b9e10 (5 days ago), 2021-02-03 15:33:39 -0800
    • Engine revision 2c527d6c7e
    • Dart version 2.12.0 (build 2.12.0-259.8.beta)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.
[!] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.
[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
[✓] VS Code (version 1.53.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0
[✓] Connected device (3 available)
    • Taha’s iPad (mobile) • 00008020-000255113EE8402E • ios            • iOS 14.4
    • macOS (desktop)      • macos                     • darwin-x64     • macOS 11.2 20D64 darwin-x64
    • Chrome (web)         • chrome                    • web-javascript • Google Chrome 88.0.4324.146
! Doctor found issues in 2 categories.
```
```bash
[✓] Flutter (Channel master, 1.26.0-18.0.pre.212, on macOS 11.2 20D64 darwin-x64, locale en-GB)
    • Flutter version 1.26.0-18.0.pre.212 at /Users/tahatesser/Code/flutter_master
    • Framework revision 02d441ea55 (2 days ago), 2021-02-05 17:17:12 -0800
    • Engine revision b04955656c
    • Dart version 2.13.0 (build 2.13.0-0.0.dev)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.
[!] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.
[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
[✓] VS Code (version 1.53.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0
[✓] Connected device (3 available)
    • Taha’s iPad (mobile) • 00008020-000255113EE8402E • ios            • iOS 14.4
    • macOS (desktop)      • macos                     • darwin-x64     • macOS 11.2 20D64 darwin-x64
    • Chrome (web)         • chrome                    • web-javascript • Google Chrome 88.0.4324.146
! Doctor found issues in 2 categories.
```

</details>

Thank you

**Comment by TahaTesser** [comment]
@Bobbbay 
Add a thumbs up reaction to https://github.com/flutter/flutter/issues/75473#issue-801796225

**Comment by kaumudpa** [comment]
I checked  - This is working fine at this commit `63062a64432cce03315d6b5196fda7912866eb37 ` or flutter version `1.26.0-1.0.pre` Something broke it somewhere after this commit.

**Comment by kaumudpa** [comment]
@yjbanov You have closed this issue but i just switched to master and checked - The problem is still there!

**Comment by kaumudpa** [comment]
@TahaTesser & @yjbanov @emmett-deen Can you kindly check and conform if you are also facing the same problem?

**Comment by emmett-deen** [comment]
@kaumudpa Can confirm still having this issue on both master & dev branches. @TahaTesser @yjbanov could we reopen this issue please?

**Comment by lorr756** [comment]
T @

**Comment by TahaTesser** [comment]
The fixed is landed and I also can't reproduce the latest `master`, also confirmed in https://github.com/flutter/flutter/issues/76186#issuecomment-780412857

<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel master, 1.27.0-5.0.pre.49, on macOS 11.2.1 20D74 darwin-x64, locale en-GB)
    • Flutter version 1.27.0-5.0.pre.49 at /Users/tahatesser/Code/flutter_master
    • Framework revision 23e41734e5 (2 hours ago), 2021-02-18 00:46:02 -0800
    • Engine revision 6993cb229b
    • Dart version 2.13.0 (build 2.13.0-30.0.dev)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.
[!] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.
[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
[✓] VS Code (version 1.53.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0
[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 11.2.1 20D74 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 88.0.4324.182
! Doctor found issues in 2 categories.
```

</details>

**Comment by imaNNeoFighT** [comment]
I confirm it has been fixed in the master channel (2.1.0-11.0.pre.145). And [fl_chart](https://github.com/imaNNeoFighT/fl_chart) works properly.

**Comment by neliousness** [comment]
its working on working on the latest version of the stable channel (Flutter 2.0.3)
